telegraf/ucs_traffic_monitor.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/python3
 
 __author__ = "Paresh Gupta"
-__version__ = "0.45"
+__version__ = "0.45.1"
```

Comment:
Please change version to 0.46

Reply:
I did bump the version to 0.46 as requested